server/parsec/asgi/rpc.py

```diff
@@ -428,8 +428,26 @@ async def anonymous_api(raw_organization_id: str, request: Request) -> Response:
     except ValueError:
         _handshake_abort_bad_content(api_version=parsed.settled_api_version)
 
+    client_ctx.logger.debug(
+        "RPC start",
+        req=req,
+    )
+
     cmd_func = backend.apis[type(req)]
     rep = await cmd_func(client_ctx, req)
+
+    if backend.config.log_level_is_debug:
+        client_ctx.logger.info(
```

Comment:
Shoudn't this be `logger.debug` instead?
```suggestion
        client_ctx.logger.debug(
```
This is what you did in the `event` component. Or use `logger.info` in the `event` component for consistency.

Reply:
:+1: